Sources/SpelledPitch/PitchSpeller/Wetherfield/Pitch.Spelling.Category.swift

```diff
@@ -10,7 +10,7 @@ import Math
 import Pitch
 
 /// Interface for the six pitch spelling categories.
-protocol PitchSpellingCategoryProtocol {
+protocol PitchSpellingCategoryProtocol: TendencyConverting {
```

Comment:
I initially wanted these to be orthogonal concepts (`PitchSpellingCategoryProtocol` and `TendencyConverting`). But that resulted in some weird awkwardness that is now cleaned up downstream as a result of this change.

So, I'm all for it. Maybe down the line we can break it up into more modular pieces if the necessity actually emerges.

Reply:
Yeah this was probably one of those reckless hacking moments so that things would work with the `Bimap`s I needed to implement. Sorry about messing with the API vision!